spec/forms/register_nonprofit_form/stripe_account_form_spec.rb

```diff
@@ -1,61 +1,54 @@
 # License: AGPL-3.0-or-later WITH Web-Template-Output-Additional-Permission-3.0-or-later
 require "rails_helper"
 
-RSpec.describe RegisterNonprofitForm::StripeAccountForm, :type => :model do
-
-  around(:each) do |ex|
+RSpec.describe RegisterNonprofitForm::StripeAccountForm, type: :model do
+  around do |ex|
     StripeMockHelper.mock do
       ex.run
     end
   end
 
-  context 'validation' do
-    it 'accepts a nonprofit' do
+  describe "validation" do
+    it "accepts a nonprofit" do
       form = described_class.new(nonprofit: build(:nonprofit))
       expect(form).to be_valid
     end
 
-    it 'rejects a string' do
+    it "rejects nil" do
       form = described_class.new(nonprofit: nil)
-      expect(form).to_not be_valid
+      expect(form).not_to be_valid
     end
 
-    it 'rejects a string' do
+    it "rejects a string" do
       form = described_class.new(nonprofit: "a string")
-      expect(form).to_not be_valid
+      expect(form).not_to be_valid
     end
   end
 
-
-  describe "#save" do 
+  describe "#save" do
     it "sets error when nonprofit is not persisted" do
       form = described_class.new(nonprofit: build(:nonprofit))
       expect(form.save).to be false
```

Comment:
Even though you didn't change this, it should honestly be `eq`. But it doesn't matter much in this case probably.